docs/networking/entities.md

```diff
@@ -0,0 +1,40 @@
+Entities
+========
+
+In addition to regular network messages, there are various other systems provided to handle synchronizing entity data.
+
+Spawn Data
+----------
+
+In general, the spawning of modded entities is handled seperately, by Forge.
+
+!!! note
+    This means that simply extending a vanilla entity class may not inherit all its behaviour here. You may need to implement certain vanilla behaviours yourself.
+
+You can add extra data to the spawn packet Forge sends by implementing the following interfaces.
+
+### IEntityAdditionalSpawnData
+
+If your entity has data that is needed on the client, but doesn't change over time, then it can be added to the entity spawn packet using this interface. `writeSpawnData` and `readSpawnData` control how the data should be en/decoded to/from the network buffer, in a similar fashion to the `toBytes`/`fromBytes` methods in `IMessage`.
```

Comment:
maybe add parentheses to the method names so it's clear that it's a method and not a field
```suggestion
If your entity has data that is needed on the client, but doesn't change over time, then it can be added to the entity spawn packet using this interface. `IEntityAdditionalSpawnData#writeSpawnData()` and `IEntityAdditionalSpawnData#readSpawnData()` control how the data should be en/decoded to/from the network buffer, in a similar fashion to the `IMessage#toBytes()`/`IMessagefromBytes()` methods.
```